introduction.md

```diff
@@ -1 +1,2 @@
 I'm Sarah. Learning to use git. I like th iOS path.
+* My deadly skins are yet to be seen.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?